code/game/objects/items/storage/storage.dm

```diff
@@ -412,10 +413,19 @@
 	return 1
 
 //Call this proc to handle the removal of an item from the storage item. The item will be moved to the atom sent as new_target
-/obj/item/storage/proc/remove_from_storage(obj/item/I, atom/new_location)
+/obj/item/storage/proc/remove_from_storage(obj/item/I, atom/new_location, mob/user)
 	if(!istype(I))
 		return FALSE
 
+	var/removal_good = TRUE
```

Comment:
Unless i'm missing something, you don't need that var. you can put the rest of the code right above the return.